Makefile

```diff
@@ -18,6 +18,11 @@ CLOUDCAFE ?= $(shell which cafe-runner)
 
 .PHONY: targets env-precheck
 
+hooks:
+	echo "#!/bin/bash" > .git/hooks/pre-commit
+	echo "python `pwd`/scripts/config_check.py" >> .git/hooks/pre-commit
+	echo "`pwd`/scripts/python-lint.py --git-precommit" >> .git/hooks/pre-commit
```

Comment:
Removed `python-lint.py` and now `config_check.py` is copied to the git hooks directory and run from there.